src/main/java/org/galatea/starter/entrypoint/IexRestController.java

```diff
@@ -20,7 +21,7 @@
 @Validated
 @RestController
 @RequiredArgsConstructor
-public class IexRestController {
+public class IexRestController extends BaseRestController {
```

Comment:
Why do we need to extend the BaseRestController class. Not that I think we shouldn't, just wondering why we did for this PR

Reply:
This was unnecessary, I'll undo it.